.azure_pipelines/ci-pipeline-code-coverage-nightly.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
         sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
         sudo apt-get update
-        sudo apt-get install build-essential lcov python3-setuptools python3-pip llvm-7 libmbedtls-dev docker-ce lldb-10 -y
+        sudo apt-get install build-essential lcov python3-setuptools python3-pip llvm-7 libmbedtls-dev docker-ce -y
```

Comment:
Is the removal of lldb-10 intended?

Reply:
Yes, its being removed from the pipeline VM setup script. The PR moves the SOS test to be run inside a docker container. The docker image has lldb-10 installed.